### PR TITLE
Fix method doc signatures for multiline required parameters

### PR DESCRIPTION
**Pull request by tcoffee-google**
Existing doc generator failed to recognize parameters as required when parameter descriptions extended over more than one line. Besides presenting incorrect information, resulting inconsistencies broke checks for automated sample generation.

### CODE REVIEW

**Comment by tcoffee-google** [comment]
Note for review: manual changes in first commit, autogenerated doc update in second commit.

**Comment by tcoffee-google** [comment]
I lack permissions to assign reviewers ... @jonparrott @saicheems please take a look. CC @vchudnov-g

File: docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html [hunk 1]
```diff
@@ -105,11 +105,11 @@
     "clientAccountId": "A String", # Numerical account ID of the client buyer
         # that the invited user is associated with.
         # The value of this field is ignored in create operations.
+    "invitationId": "A String", # The unique numerical ID of the invitation that is sent to the user.
```
Comment (saicheems) [inline docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html]: Unrelated:

Just realized this was nondeterministic, how hard would it be to have a fixed ordering?

**Comment by jonparrott** [comment]
LGTM, let me know when you want to merge.

**Comment by tcoffee-google** [comment]
@jonparrott Please go ahead and merge; I will investigate the ordering issue @saicheems brings up (to reduce diff noise, I presume).

File: docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html (hunk 1 above)
Comment (tcoffee-google) [inline docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html]: @saicheems Sorry I have not had a chance to investigate this further; filed https://github.com/google/google-api-python-client/issues/386.

**Pull request merged**